Don't build pk3 files because they already exist.
Also do not remove them on clean.
Index: ufoai-2.4/build/data.mk
===================================================================
--- ufoai-2.4.orig/build/data.mk	2012-04-22 23:01:29.000000000 +0200
+++ ufoai-2.4/build/data.mk	2012-04-23 19:37:55.317342282 +0200
@@ -3,10 +3,10 @@
 BASE_DIR = base
 PAK_FILES_OUT = $(addprefix $(BASE_DIR)/,$(PAK_FILES))
 
-pk3: $(PAK_FILES_OUT)
+pk3:
 
 clean-pk3:
-	$(Q)rm -f $(PAK_FILES_OUT)
+#	$(Q)rm -f $(PAK_FILES_OUT)
 
 define FIND
 $(shell find $(BASE_DIR)/$(1) -type f -print)
